servicecontrol/logging.md

```diff
@@ -10,18 +10,71 @@ redirects:
 
 ### Logging
 
-Instances of the ServiceControl service write logging information and failed message imports to the file system.  
-Before v1.9 the logging level by default was set to `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding the following to the configuration file:
+Instances of the ServiceControl service write logging information and failed message import stack traces to the file system.  
+
+
+#### Version 1.8.3 and below 
+
+The default logging level was `Info`, 
+
+#### Version 1.9 and above
+
+The default logging level is `Warn`, this level is now configurable by adding the following to the `appSettings` section of the  configuration file:
 
 ```xml
 <!-- Log Level Options: Trace, Debug, Info, Warn, Error, Fatal, Off -->
 <add key="ServiceControl/LogLevel" value="Info" /> 
 ```
 
-NOTE: If ServiceControl experiences a critical exception when running as a service the exception information will be logged to the Windows EventLog rather than the log file.
+### RavenDB Logging 
+
+#### Version 1.9 and below
+
+RavenDB logging is included in the ServiceControl logs.  This logging is hard coded to `ERROR` and above and is not affected by the "ServiceControl/LogLevel" configuration setting.
+
+#### Version 1.10 and above
+
+The RavenDB logging has been separated out into it's own log files.  These logs are located in the same directory as ServiceControl logs.  The default logging level for the RavenDB log is `Warn`.
+
+The log level for the RavenDB Logs can be set by adding the following to the `appSettings` section of the configuration file:
+
+```xml
+<!-- Log Level Options: Trace, Debug, Info, Warn, Error, Fatal, Off -->
+<add key="ServiceControl/RavenDBLogLevel" value="Info" /> 
+```
 
+### Log File Names and Retention
 
-### Location
+#### Version 1.9 and below 
+
+The current log file is named `logfile.txt`.  
+Whwn rolled logfiles are named `log.<sequencenumber>.txt`
+The sequence number starts at 0.  Lower numbers indicate more recent logfiles.
+
+ServiceControl will retain 14 logs files, older logs are deleted automatically.
+
+#### Version 1.10 and above
+
+The current ServiceControl log file is named `logfile.<data>.txt`
+The current RavenDB embedded log file is named `ravenlog.<date>.txt`.
+  
+
+The date is written in the `yyyy-MM-dd` format.
+When the log is rolled based on size the file name will contain a sequence number after the date. e.g `logfile.2016-01-16-1.txt`
+The sequence number starts at 0.  Lower numbers indicate more recent log files. 
+ServiceControl will retain 14 logs files, older logs are deleted automatically.
+
+NOTE: The change in log naming will result in logs produced prior to Version 1.10 being ignored by the log cleanup process.  These old logs can safely be removed manually.  
```

Comment:
no worries, to me it just feels wrong from an english grammar perspective, hence the reason I picked on it